python/src/wslink/server.py

```diff
@@ -220,6 +224,8 @@ def start_webserver(options, protocol=wsl.ServerProtocol, disableLogging=False):
     # Initialize testing: checks if we're doing a test and sets it up
     # testing.initialize(options, reactor, stop_webserver)
 
+    reactor.callWhenRunning(print_ready)
```

Comment:
You may know more about it than I do, but it seems to me that once the reactor is running, we should be good to go:

https://twistedmatrix.com/documents/current/api/twisted.internet.interfaces.IReactorCore.html#callWhenRunning

We used the hook above to know whether our test thread should try to connect to the running server, and that seemed to be reliable.